packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -192,8 +195,11 @@ class CircleAvatar extends StatelessWidget {
   Widget build(BuildContext context) {
     assert(debugCheckHasMediaQuery(context));
     final ThemeData theme = Theme.of(context);
-    TextStyle textStyle = theme.primaryTextTheme.titleMedium!.copyWith(color: foregroundColor);
-    Color? effectiveBackgroundColor = backgroundColor;
+    final Color? effectiveForegroundColor = foregroundColor
+      ?? (theme.useMaterial3 ? theme.colorScheme.onPrimaryContainer : null);
+    TextStyle textStyle = theme.primaryTextTheme.titleMedium!.copyWith(color: effectiveForegroundColor);
```

Comment:
We should be using `theme.textTheme` here. primaryTextTheme, like primaryColorLight and primaryColorDark, are obsolete and will be deprecated when Material3 becomes the default.

Reply:
This seems to be the M2 default color. I will look into it.


![Screenshot 2022-11-08 at 18 43 39](https://user-images.githubusercontent.com/48603081/200624588-d18d7dbd-7613-44fe-b99b-338160960ce2.png)


![Screenshot 2022-11-08 at 18 43 26](https://user-images.githubusercontent.com/48603081/200624581-b578e345-a462-40ff-ad3b-785e4c71b728.png)